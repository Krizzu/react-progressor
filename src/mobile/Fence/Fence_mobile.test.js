import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import FenceProgressor from './Fence';

it('Renders correctly', () => {
  const tree = renderer.create(
    <FenceProgressor />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
