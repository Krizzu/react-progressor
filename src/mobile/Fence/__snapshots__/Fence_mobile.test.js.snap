// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-end",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "position": "relative",
      },
      Object {
        "height": 30,
        "width": 200,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#eee",
          "height": 20,
          "width": 18.181818181818183,
        },
        undefined,
      ]
    }
  />
</View>
`;
